packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -480,7 +481,12 @@ export const ImageInput = ({
           <LoadingLabel>{loadingLabel}</LoadingLabel>
         </LoadingIcon>
       </InputWrapper>
-      <ValidationHint validationHint={validationHint} invalid={invalid} />
+      <ValidationHint
+        id="validation_hint"
+        aria-live="assertive"
```

Comment:
As far as I know, it's recommended to add the live region to a wrapping HTML element that already exists in the DOM _before_ injecting the error message. Otherwise, some SR/browser combinations might not announce it properly